server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessagesMethodTest.java

```diff
@@ -331,4 +331,34 @@ public void getMessagesShouldReturnMandatoryPropertiesMessagesWhenNotAsked() thr
             .body(ARGUMENTS + ".list[0].id", equalTo("username@domain.tld|mailbox|1"))
             .body(ARGUMENTS + ".list[0].subject", equalTo("my test subject"));
```

Comment:
Tests with a message without attachements ? (sure you can add this check to an existing test)